scarab_stats/scarab_stats.py

```diff
@@ -1575,7 +1579,24 @@ def load_simpoint_data(self, cluster_id, workload, config, experiment_name, arch
     args = parser.parse_args()
 
     da = stat_aggregator()
-    E = da.load_experiment_json(args.descriptor_name, True)
+    
+    # E = da.load_experiment_json(args.descriptor_name, True)
+    # E.to_csv("micro.csv")
+
+    E = da.load_experiment_csv("micro.csv")
```

Comment:
I think we do not always want to load micro.csv
please check the other lines below as well.